packages/semcom-demo/src/app/connect/services/provider.service.ts

```diff
@@ -0,0 +1,16 @@
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { Provider } from '../models/provider.model';
+
+@Injectable({ providedIn: 'root' })
+export class ProviderService {
+
+  getAll(): Observable<Provider[]> {
+    return new Observable(subscriber => subscriber.next([
```

Comment:
You could use the `of` operator to simplify this a little.

https://www.learnrxjs.io/learn-rxjs/operators/creation/of